README.md

```diff
@@ -19,10 +19,9 @@
 
 ## Maintainers
 
-| Maintainer    | GitHub                                      | Social                                           |
-| ------------- | ------------------------------------------- | ------------------------------------------------ |
-| Priyank Patel | [priyankpat](https://github.com/priyankpat) | [@priyankpat\_](https://twitter.com/priyankpat_) |
-| Stewan Silva  | [stewwan](https://github.com/stewwan)       | [@StewanSilva](https://twitter.com/StewanSilva)  |
+| Maintainer      | GitHub                                |
+| --------------- | ------------------------------------- |
+| Gustavo Horisch | [gugahoi](https://github.com/gugahoi) |
```

Comment:
Typo in your last name, no need to fix just thought I'd flag

Reply:
🤦